recipes/pymrio/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pymrio" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymrio-{{ version }}.tar.gz
+  sha256: d8d96516fcf2b8dd2c2d60ef9443630a6774c21495f98d2bed8b0c53883c7718
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - matplotlib-base >=2.0.0
+    - numpy >=1.13.4
+    - pandas >=0.25.0
+    - python >=3.7
+    - requests >=2.18
+    - xlrd >=1.1.0
+    - openpyxl >=3.0.4
+
+test:
+  imports:
+    - pymrio
+    - pymrio.core
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/konstantinstadler/pymrio
+  summary: A python module for automating input output calculations and generating reports
+  license: GPL-3.0-or-later
+  license_file: LICENSE
```

Comment:
I've opened a PR upstream to try to get the license file included in the source distribution (https://github.com/konstantinstadler/pymrio/pull/56). If you could add the following comment in the `meta.yaml`:

```yaml
# License file manually packaged. See: https://github.com/konstantinstadler/pymrio/pull/56
license_file: LICENSE
...
```

This will be a visual reminder to check if the license file is in the source distribution in future releases when you update the feedstock. If it is, then you can remove the comment and the manually packaged copy of the license.